zivid_camera/src/zivid_camera.cpp

```diff
@@ -240,6 +240,8 @@ ZividCamera::ZividCamera(ros::NodeHandle& nh, ros::NodeHandle& priv)
       nh_.advertiseService("camera_info/serial_number", &ZividCamera::cameraInfoSerialNumberServiceHandler, this);
   is_connected_service_ = nh_.advertiseService("is_connected", &ZividCamera::isConnectedServiceHandler, this);
   capture_service_ = nh_.advertiseService("capture", &ZividCamera::captureServiceHandler, this);
+  capture_and_save_frame_service_ =
+      nh_.advertiseService("capture_and_save_frame", &ZividCamera::captureAndSaveFrameServiceHandler, this);
```

Comment:
We also discussed just capture_and_save, did you prefer keeping frame? Another slightly more verbose option is capture_and_save_to_file

Reply:
`capture_and_save` can open up questions about what? Same with `capture_and_save_to_file`.